.github/workflows/main_housingmanagementsystemapi.yml

```diff
@@ -23,6 +23,11 @@ jobs:
         run: dotnet tool update --global dotnet-format
       - name: Run dotnet format
         run: dotnet format --check --verbosity diagnostic
+      - uses: hashicorp/setup-terraform@v2
+      - name: Terraform fmt

```

Comment:
We would want each format job to run every time regardless of if the previous had failed.
The current configuration would mean if the `Run dotnet format` step fails, then `Terraform fmt` wouldn't be run.

This [SO answer](https://stackoverflow.com/a/58859404) and the [official documentation](https://docs.github.com/en/actions/learn-github-actions/expressions#always) may help in always running each step

Reply:
Done in [8ba50d0](https://github.com/Newarkandsherwood/HousingManagementSystemApi/pull/9/commits/8ba50d0b85e67675434435ea1a1df0b802236544)